web_src/js/utils/dom.test.js

```diff
@@ -0,0 +1,6 @@
+import {createElementFromHTML} from './dom.js';
+
+test('createElementFromHTML', () => {
+  expect(createElementFromHTML('<a>foo<span>bar</span></a>').textContent).toEqual('foobar');
```

Comment:
```suggestion
```

Since it has fully tested the `outerHTML`, then it doesn't need to test `textContent` again.